google/cloud/storage/internal/bucket_requests_test.cc

```diff
@@ -15,6 +15,10 @@
 #include "google/cloud/storage/internal/bucket_requests.h"
 #include <gmock/gmock.h>
 
+namespace google_cloud_storage_internal_nlohmann_3_1_2 {
+inline void PrintTo(json const& json, std::ostream* os) { *os << json.dump(); }
```

Comment:
Is this actually more helpful than `os << json.dump()`?  I also don't see this used anywhere - did you add it and forget it remove it later, maybe?